core/src/main/java/com/linecorp/armeria/server/docs/DocService.java

```diff
@@ -178,4 +165,31 @@ void setSpecification(byte[] content) {
             entry = new ByteArrayEntry("/", MediaType.JSON_UTF_8, content);
         }
     }
+
+    private static final class ServiceExamplesImpl implements ServiceExamples {
+        private final ListMultimap<String, HttpHeaders> exampleHttpHeaders;
+        private final ListMultimap<String, String> exampleRequests;
+
+        ServiceExamplesImpl(ListMultimap<String, HttpHeaders> exampleHttpHeaders,
+                            ListMultimap<String, String> exampleRequests) {
+            this.exampleHttpHeaders = exampleHttpHeaders;
+            this.exampleRequests = exampleRequests;
+        }
+
+        @Override
+        public List<HttpHeaders> exampleHttpHeaders(String serviceName) {
+            requireNonNull(serviceName, "serviceName");
+            return ImmutableList.<HttpHeaders>builder()
+                                .addAll(exampleHttpHeaders.get(serviceName))
+                                .addAll(exampleHttpHeaders.get(""))
+                                .build();
+        }
+
+        @Override
+        public List<String> exampleRequests(String serviceName, String functionName) {
+            requireNonNull(serviceName, "serviceName");
+            requireNonNull(functionName, "functionName");
+            return exampleRequests.get(serviceName + '#' + functionName);
```

Comment:
I thought it'd just be ```String``` - row serviceName column functionName. Then this separator ```#``` goes away and it's more structured.